tests/acceptance/bootstrap/SpacesContext.php

```diff
@@ -74,6 +74,27 @@ class SpacesContext implements Context {
 	 */
 	private array $storedEtags = [];
 
+	/**
+	 * @param string $spaceCreator
+	 * @param object $space
+	 *
+	 * @return void
+	 */
+	public function addToCreatedSpace(string $spaceCreator, object $space): void {
```

Comment:
should we should update created space too? we have feature to update spaces

Reply:
currently, we use spaces store to get only the id and space-creator. so other information are not used yet. update store method can be added when it's required.